src/todo.txt

```diff
@@ -0,0 +1,25 @@
+stuff to expose:
```

Comment:
This TODO file is a list of ideas I still have to go through and decide for this to be an usable library.